### PR TITLE
ETH-658: Fixed early leaver penalty

### PR DESCRIPTION
**Pull request by jtakalai**
early leaver penalty used to depend on stake, now it's flat X DATA for everyone

add earlyLeaverPenaltyWei to StreamrConfig, use it in DefaultLeavePolicy

fixed tests, some to use real StreamrConfig values

### CODE REVIEW

**Comment by hpihkala** [comment]
Assuming ready for review since this is not a draft PR. Requested review as Juuso might have forgotten to click that rushing to his holiday :)

**Comment by jtakalai** [comment]
uh well, this was just the beginning of the PR really; since we also want unit tests... so yea more like a draft PR I guess

File: packages/network-contracts/test/hardhat/OperatorTokenomics/Operator.test.ts [hunk 1]
```diff
@@ -653,17 +653,17 @@ describe("Operator contract", (): void => {
...
+                .to.emit(operator, "Loss").withArgs(parseEther("1000"))
+                .to.emit(operator, "OperatorSlashed").withArgs(parseEther("1000"), parseEther("1000"), parseEther("1000"))
 
             // leave penalty goes to the protocol
-            expect(formatEther(await token.balanceOf(protocolFeeBeneficiary.address))).to.equal("100.0")
+            expect(formatEther(await token.balanceOf(protocolFeeBeneficiary.address))).to.equal("1000.0")
         })
 
-        it("if operator has no self-delegation, it won't get slashed for losses either", async function(): Promise<void> {
+        it.only("if operator has no self-delegation, it won't get slashed for losses either", async function(): Promise<void> {
```
Comment (samt1803) [inline packages/network-contracts/test/hardhat/OperatorTokenomics/Operator.test.ts:666]:
```suggestion
        it("if operator has no self-delegation, it won't get slashed for losses either", async function(): Promise<void> {
```
re-enable all tests

**Review by samt1803 — CHANGES_REQUESTED** [review]
some tests still fail when re-enabling all tests

**Review by samt1803 — APPROVED** [review]

**Pull request merged**